make_dataset.py

```diff
@@ -1,13 +1,13 @@
 from utils.data_utils import generate_airbnb_copies, generate_communities_copies, save_downsample_communities, save_downsample_airbnb
-# generate_airbnb_copies(100)
-# generate_airbnb_copies(250)
-# generate_communities_copies(8)
-# save_downsample_communities(20000)
-# save_downsample_communities(5000)
-# save_downsample_communities(50000)
+generate_airbnb_copies(100)
+generate_airbnb_copies(250)
+generate_communities_copies(100) # assumes lux-datasets is in the same directory as lux-benchmark
```

Comment:
Let's add scripts here to automatically create a `data/` directory and download the files into that directory if it does not already exist.